docs/running-on-mesos.md

```diff
@@ -469,6 +470,15 @@ See the [configuration page](configuration.html) for information on Spark config
   </td>
 </tr>
 <tr>
+  <td><code>spark.mesos.driver.labels</code></td>
```

Comment:
This naming differs a bit from the YARN label property, but I suppose it's supporting an expression. And we have .task.labels already. OK.